### PR TITLE
Add Drift

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Review by ApCoder123 — COMMENTED** [review]
Please could you provide some evidence that 2FA is offered. 
The doc page linked just states that all Drift employees use 2FA, with no mention of 2FA being available for Drift users.

> Everyone on our team has two-factor authentication setup for their accounts where possible, which means that nobody can access their accounts without a security code sent to our mobile devices at the time of attempted login.

**Comment by 40504** [comment]
Ups, I forgot to attach the screenshot 🤦🏻‍♂️

**Review by Carlgo11 — APPROVED** [review]

File: _data/other.yml [hunk 1]
```diff
@@ -194,6 +194,13 @@ websites:
     img: dreamstime.png
     twitter: Dreamstime
 
+  - name: Drift
+    url: https://www.drift.com/
+    img: drift.svg
+    tfa:
+      - totp
+    doc: https://gethelp.drift.com/hc/en-us/articles/360019504214
```
Comment (ApCoder123) [inline _data/other.yml]:
```suggestion

```
Probably best to remove this doc link - it doesn't tell a user any useful info about 2FA on their account

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**